### DISCUSSION TITLE
Is MirrorMaker2 restarted when applying Strimzi MM2 config after topics whitelist change?

### DISCUSSION THREAD

**Discussion by EmiliaNowakowska** [strimzi/strimzi-kafka-operator#6459] [Q&A] [answered]
Hello, 

I'm using Strimzi MirrorMaker2 for replication and there is a need to update the topics whitelist and consumer group whitelist quite often. 
Could you please tell me if MirrorMaker2 is restarted each time after applying the yaml config file with changed whitelist? 
I'm still not sure about it after pods logs review. 
Could you please tell me how it works, is MM2 restarted, is the replication paused when we change only the whitelist values and apply the config?

Thank you in advance for your answer. 
BR, 
Emilia

**Comment by scholzj** [comment, marked as answer]
Strimzi runs MM2 as connectors on top of Kafka Connect. When the configuration of Kafka connect itself is changed, it will need to be restarted -> you should see that the pods are being terminated and new pods are started. When you change the configuration of the MM2 connectors, they are updated using the REST API. The Pod itself will not be restarted, but the MM2 connector will be (as a _Java object_ inside the Kafka Connect).

**Reply by EmiliaNowakowska** [reply]
Thank you very much for your quick reply and explanation! 

BR,
Emilia
